package.json

```diff
@@ -23,13 +23,14 @@
   },
   "dependencies": {
     "@blockworks-foundation/mango-client": "^3.0.11",
-    "@project-serum/serum": "^0.13.20",
+    "@project-serum/serum": "^0.13.65",
     "@solana/web3.js": "^0.91.0",
     "axios": "^0.21.1",
     "cors": "^2.8.5",
     "dayjs": "^1.10.7",
     "express": "^4.17.1",
     "lru-cache": "^5.1.0",
+    "npm-run-all": "^4.1.5",
```

Comment:
Why do we need a new package here? I don’t see it used anywhere

Reply:
Good question. I wasn't able to run the script `"prepare": "run-s clean postinstall"`. run-s is part of the npm-run-all package. Couldn't we change the "prepare" script to "npm run clean && npm run postinstall" and remove run-s/npm-run-all as a dependency?